Batteries/Data/List/Basic.lean

```diff
@@ -736,7 +736,15 @@ def sigmaTR {σ : α → Type _} (l₁ : List α) (l₂ : ∀ a, List (σ a)) :
 ofFn f = [f 0, f 1, ... , f (n - 1)]
 ```
 -/
-def ofFn {n} (f : Fin n → α) : List α := (Array.ofFn f).data
+def ofFn {n} (f : Fin n → α) : List α := go n (Nat.le_refl _) where
+  /-- Auxillary for `List.ofFn`. -/
```

Comment:
```suggestion
  /-- Auxiliary for `List.ofFn`. -/
```
I'm glad to discover that I'm not the only person who can not spell this word. :-)